### DISCUSSION TITLE
Is it possible to install nano on this machine somehow?

### DISCUSSION THREAD

**Discussion by 621625** [copy/v86#867] [General]
Hi,

I'm Mr. Granite, a very friendly guy.

Is it possible to install nano on this machine somehow?

https://copy.sh/v86/?profile=buildroot

**Comment by spetterman66** [comment]
can't you like, compile it on your own?

**Comment by 621625** [comment]
Possibly. But is there no way to install it using a package manager or by downloading a binary?

**Comment by spetterman66** [comment]
well yeah you could try to get gcc on there since you have networking (run `udhcpc`)

**Comment by 621625** [comment]
Ok, I might try later. I will use vi for now.

Another question: I have saved the state as v86state.bin

Is there an easy way to make a HTML page that automatically loads this particular state and only that, without including the top menu bar with all the buttons?

**Comment by spetterman66** [comment]
can't you try building your own buildroot instance with nano and x11 included (or the archlinux32 profile at http;//copy.sh/v86?profile=archlinux)?
